docs/user-guides/desktop/index.md

```diff
@@ -0,0 +1,82 @@
+# Coder Desktop
+
+Use Coder Desktop to work on your workspaces as though they're on your LAN, no
+port-forwarding required.
+
+## Install Coder Desktop
+
+<div class="tabs">
+
+You can install Coder Desktop on macOS or Windows.
+
+### macOS
+
+1. Use [Homebrew](https://brew.sh/) to install the Coder Desktop from [cask](https://github.com/coder/coder-desktop-macos/tree/main):
+
+   ```shell
+   brew install --cask coder/coder/coder-desktop-preview
```

Comment:
```suggestion
   brew install --cask coder/coder/coder-desktop
```
This should be `coder-desktop`. The `preview` version is for internal testing only.

I know it is not available, but it will be as we push the next release tag.